src/js/popup.js

```diff
@@ -247,9 +263,32 @@ function init() {
     visibility: 'visible'
   });
 
+  let link = getLink();
+  $("#cta-link").attr("href", link.url);
+  $('#cta-text').html(chrome.i18n.getMessage(link.text));
+  $('#cta-icon').addClass(link.icon);
+
   window.POPUP_INITIALIZED = true;
 }
 
+/**
+ * Get the link to display at the bottom of the popup
+ */
+function getLink() {
+  let rand = Math.random();
+  let cumulativeOdds = 0;
+
+  for (let link of linkRotation) {
+    cumulativeOdds += link.odds || 0;
```

Comment:
This is hard to read ... I don't trust the precedence is what it should be. One way to fix is with parenthesis.